openslides/motions/serializers.py

```diff
@@ -344,12 +372,25 @@ class Meta:
     def validate(self, data):
         if 'text'in data:
             data['text'] = validate_html(data['text'])
+
         if 'reason' in data:
             data['reason'] = validate_html(data['reason'])
+
         validated_comments = dict()
         for id, comment in data.get('comments', {}).items():
             validated_comments[id] = validate_html(comment)
         data['comments'] = validated_comments
+
+        if 'amendment_paragraphs' in data:
+            data['amendment_paragraphs'] = list(map(lambda entry: validate_html(entry) if type(entry) is str else None,
+                                                    data['amendment_paragraphs']))
+            data['text'] = ''
+        else:
+            if 'text' in data and len(data['text']) == 0:
+                raise serializers.ValidationError({
+                    'text': _('This field may not be blank.')
```

Comment:
I am not sure whether this string needs to be translated. Also the keyword should be 'detail' and not 'text', but I am not sure.

Reply:
Done.